robot/demos/RosArduinoBlinkLed/README.md

```diff
@@ -0,0 +1,64 @@
+# ROS Arduino Blink LED Demo
+
+## Setup
+
+All you need is any arduino (with a working built in LED), a USB-B to USB-A cable, and a [computer capable of running ROS](http://wiki.ros.org/kinetic/Installation).
+I installed ROS using the quick install script from the ROS programming book:
+
+1. `wget https://raw.githubusercontent.com/ROBOTIS-GIT/robotis_tools/master/install_ros_kinetic.sh`
+2. `chmod 755 ./install_ros_kinetic.sh`
+3. `bash ./install_ros_kinetic.sh`
+
+For setting up [Arduino IDE](https://www.arduino.cc/en/Main/Software), follow instructions [here](http://wiki.ros.org/rosserial_arduino/Tutorials/Arduino%20IDE%20Setup)
+
+Make sure you run the following commands to install rosserial libraries:
+- `sudo apt-get install ros-kinetic-serial-arduino`
+- `sudo apt-get install ros-kinetic-serial`
+
+## Running the demo
+
+### One machine
+
+Launch [roscore](http://wiki.ros.org/roscore) in a new terminal window:
+
+- `roscore`
+
+Next, run the rosserial client application that forwards your Arduino messages ti the rest of ROS.
+Make sure to use the correct serial port:
```

Comment:
I suggest saying "Make sure to verify the serial port being used on your specific computer to talk to the Arduino. On my computer it is xxxx"

Reply:
maybe overkill but you could also provide a link on how to find this info